web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.constants.ts

```diff
@@ -1 +1,5 @@
 export const SEE_ALL_METHODOLOGIES = 'See all methodologies';
+
+// transparent pixel
+export const PROJECT_STANDARD_DEFAULT_VALUE =
```

Comment:
This would probably be reusable, might be worth putting it elsewhere and rename for that purpose.

Reply:
where do you think we could put this instead?

Reply:
I would say `lib/utils/image`.

Reply:
we have some `utils` folder at the top of web-registry already, maybe there instead?

Reply:
https://github.com/regen-network/regen-web/pull/1940/commits/3d3e8a8a85412ad6dc19dc1a6def7be438837ba2